docu/Concepts/Snippets/1-Blockchain/Connector/Dezentralität.md

```diff
@@ -0,0 +1,70 @@
+## Dezentralität
+
+Ich habe von Anfang an versucht Gradido so dezentral wie möglich zu entwerfen.
+Dezentral an der Basis. Kryptografie und Blockchain-Technologie sind meiner Meinung nach die besten Technologien dafür.
+Sie ermöglichen einen Ansatz bei dem der Benutzer und seine Daten im Mittelpunkt stehen. In der Form,
+das nur der Benutzer selbst die volle Kontrolle über seine Daten haben kann. 
+Dadurch wird der Benutzer unabhägig von Server-Admins und das Gesamtsystem hat weniger Points of Failure.
+Bei den Crypto-Coins wie Bitcoin wird diese Kontrolle des Benutzers über seine Daten durch die Nutzung
+von Public-Private Kryptografie erreicht. Damit sind zwei grundlegende Arten von Operationen möglich.
+
+Die eine Art der Operationen kann nur derjenige ausführen, der Zugriff auf den Private Key hat. 
+Das Ergebnis kann jedoch von allen überprüft werden die Zugriff auf den Public Key haben. 
+
+Und es gibt Operationen die man mit dem public key ausführen kann, bei denen nur der Inhaber das Private Keys das 
+Ergebnis sehen kann. 
+
+Wenn mehr Wert auf Anonymität gelegt wird, gibt es auch sogenannte Zero-Knowledge Protokolle wie bei zcash welche Transaktionen ermöglichen 
+die nicht nachvollziehbar sind. 
+
+In der Praxis heißt das, das jede Transaktion die von einem Nutzer kommt mit dessen Private Key signiert wird. 
+Dabei fließen alle relevanten Daten der Transaktion mit in die Signatur ein und können daher später überprüft werden ob sie 
+verändert wurden. Denn eine neue gültige Signatur kann nur derjenige erstellen welche Zugriff auf den Private Key hat. 
+Mit dem Private Key steht und fällt alles. Daher ist es wichtig das dieser gut geschützt ist. 
+
+Diese Transaktionsblöcke werden dann in Form einer Blockchain gespeichert. Sie werden durch einen running hash oder transactions hash 
+miteinander verbunden. Dieser wird aus dem vorherigen Hash, der Transaktionsnr, Empfangsdatum und der Signaturen berechnet. 
+Jeder der einen Node-Server betreibt hat dadurch eine einfache Möglichkeit zu überprüfen ob andere Node-Server einwandfrei funktionieren,
+indem er einfach den running hash von einer Transaktion auf seinem Node mit dem running hash derselben Transaktion (gleiche nr) des entsprechenden 
+Node-Servers vergleicht.
+
+### Wo werden nun die Daten genau gespeichert?
+Bei Bitcoin und Co. werden einfach alle Transaktionen bei allen Node Betreibern gespeichert. Das stößt aber schnell an physikalische Grenzen. 
+Daher werden die Daten bei Gradido in spearate Blockchains pro Community gespeichert. Dabei ist der Node-Server so ausgelegt, das er beliebig viele
+Blockchains von Communities abspeichern kann. Das hängt letzendlich von der Leistung der Hardware ab auf der er läuft und wie groß und aktiv die jeweiligen Communities sind. 
```

Comment:
Does this mean the node server has to hold all blockchains it interacts with? If so how is this an significant advantage over a single blockchain?
How can between-blockchain-transactions be judged regarding tamper proof and all other abilities of blockchains? Do they apply or do they not apply? Does availability of a foreign blockchain affect temper-proof-ness?